bin/start-nginx

```diff
@@ -28,30 +28,6 @@ echo 'buildpack=nginx at=logs-initialized'
 	echo 'logs' >$psmgr
 ) &
 
-#Start App Server
```

Comment:
what is this code removal about?

Reply:
This NGINX buildpack is supposed to be used for the proxy-only. This means it won't be running stuff like unicorn/puma or whatever, it will run standalone so there is no such thing as an App Server.